src/main/java/ui/issuepanel/PanelControl.java

```diff
@@ -39,6 +45,8 @@ public PanelControl(UI ui, Preferences prefs) {
                 setCurrentlySelectedPanel(Optional.of(e.panelIndex)));
         ui.registerEvent((PanelClickedEventHandler) e ->
                 setCurrentlySelectedPanel(Optional.of(e.panelIndex)));
+        ui.registerEvent((ShowRenamePanelEventHandler) e -> Platform.runLater(() ->
+                ((FilterPanel) getPanel(e.panelId)).showRenameTextField()));
```

Comment:
The tests do trigger it, to prevent ```NoNodesFoundException``` on Travis. I think with ```mainStage.show()``` moved to UI, the event handler can likely be removed again without causing the exception. Shall I do it?